celery_slack/utils.py

```diff
@@ -0,0 +1,12 @@
+def ignore(*exceptions):
+    """
+
+    :type exceptions: typing.List[Exception]
+    """
+    def wrapper(f):
```

Comment:
Let's use [functools.wraps](https://docs.python.org/3/library/functools.html#functools.wraps) here as we do in `callbacks.py`.